.github/workflows/pull-request.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         if: steps.changed-files.outputs.only_changed != 'true'
         env: 
           CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
-        run: npx eslint ${CHANGED_FILES} && python .github/workflows/eslint_disable_check.py
```

Comment:
This file is out of scope of the PR and must be excluded from the submission.

Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.